analyzer/util/util.go

```diff
@@ -93,3 +95,11 @@ func CurrentBound(cs staking.CommissionSchedule, now beacon.EpochTime) (currentB
 
 	return latestStartedStep, uint64(cs.Bounds[i].Start - 1)
 }
+
+func SanitizeTxHash(hash string) string {
+	if hash == EmptyTxHash {
+		return "NULL"
```

Comment:
I would be surprised if this eventually does anything other than insert the actual string `"NULL"` into the db as the `tx_hash`. Not sure how pgx works, but my guess is you can consume and return `*string` in this func, and set it to `nil` when suitable.

Edit: [yup](https://stackoverflow.com/questions/63785376/inserting-empty-string-or-null-into-postgres-as-null-using-jackc-pgx)

Reply:
Ah, `nil` is the right thing then.